### PR TITLE
Feature/socket io rooms

### PR DESCRIPTION
**Pull request by nikitaprodanov**
Predefined rooms for public chating with lobby. No styling only functionality. Will use as a base for the private rooms for the actual game

### CODE REVIEW

**Comment by velisav-boyanov** [comment]
Great work colleague.

**Review by velisav-boyanov — APPROVED** [review]

File: static/scripts/socketio.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+document.addEventListener('DOMContentLoaded', () => {
+	var socket = io.connect('http://' + document.domain + ':' + location.port);
```
Comment (samigeorgiev) [inline static/scripts/socketio.js:2]: use let or const. var is strongly deprecated

File: static/scripts/socketio.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+document.addEventListener('DOMContentLoaded', () => {
```
Comment (samigeorgiev) [inline static/scripts/socketio.js:1]: you could move your script import at the bottom of the body or add 'defer' as a script property - <script defer></script>

File: static/scripts/socketio.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	var socket = io.connect('http://' + document.domain + ':' + location.port);
+
+	let room = "Lobby";
```
Comment (samigeorgiev) [inline static/scripts/socketio.js:4]: just as here

**Pull request merged**